### DISCUSSION TITLE
Information Centrality

### DISCUSSION THREAD

**Discussion by yifei-liu-yl** [networkx/networkx#5946] [Q&A]
Hi Networkx discussion board,

I have network graph which I want to calculate information centrality for. I tried following the Networkx reference page on using this package but received error message.

My code is:
`
G = nx.read_edgelist('GraphEdge_2011.txt', create_using=nx.DiGraph, delimiter=",", nodetype=int)
info = nx.information_centrality(G, weight=None, dtype='float', solver='lu')
`

I got an error message: NetworkXNotImplemented: not implemented for directed type. 

My graph shouldn't have any direction. Why do I get this error message and how should I solve it?

Thanks a lot!

**Comment by brocla** [comment]
It was the parameter `create_using=nx.DiGraph` .  This defined the graph as directional.
Try,
```python
create_using=nx.Graph
```
instead.
